ogr/services/mock/core.py

```diff
@@ -0,0 +1,82 @@
+import os
+import inspect
+import yaml
+
+def write_output(*args):
```

Comment:
why not use print right away?

Reply:
Sure, I justed just wrap print command and easily replace it with some logger, or whatever